sig-etcd/vision.md

```diff
@@ -0,0 +1,100 @@
+# SIG etcd Vision
+
+The long-term success of the etcd project depends on the following:
+- Etcd the most reliable key-value storage
```

Comment:
Similarly to the "best" comment above, I'm not sure the goal might be better framed without the implied comparison to other solutions. I agree etcd should be extremely reliable -- it's one of etcd's biggest strengths and why Kubernetes relies on it. But is there a slightly different way to convey that without stating that it needs to be the absolute best?

Reply:
Done.